build/run-in-docker

```diff
@@ -50,7 +50,7 @@ if (( $# == 0 )); then
   DOCKER_OPTS="${DOCKER_OPTS} -it"
   RUN_CMD="/bin/bash"
 else
-  RUN_CMD="$*"
+  RUN_CMD="${@@Q}"
```

Comment:
nit: we would not need to build a string from an array if we used the alternative syntax https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html/packaging_guide/sect-enabling_the_software_collection
with `--` on L77

```diff
 if (( $# == 0 )); then
   # no arguments gets an interactive shell
   DOCKER_OPTS="${DOCKER_OPTS} -it"
-  RUN_CMD="/bin/bash"
+  RUN_CMD=("/bin/bash")
 else
-  RUN_CMD="${@@Q}"
+  RUN_CMD=("$@")
 fi
...
-  scl enable devtoolset-11 "$RUN_CMD"
+  scl enable devtoolset-11 -- "${RUN_CMD[@]}"
```